packages/core/package.json

```diff
@@ -47,5 +46,8 @@
     "node-fetch": "^2.6.0",
     "tar": "^6.0.2",
     "valid-url": "^1.0.9"
+  },
+  "devDependencies": {
+    "@types/mime-types": "^2.1.0"
```

Comment:
I think @andreban made a change that moved all the dev dependencies somewhere - https://github.com/GoogleChromeLabs/bubblewrap/commit/ad139d138146a3bb1a22b310db38efa1bfe0d288 .

I'm not sure if this goes against that, but lets see what he says.

Reply:
Can you revert this change, please. `mime-types` is needed as a runtime dependency. This is also downgrading the minimum version from `2.1.27` to `2.1.0`. Generally, `devDependencies` should be empty in `core`, `cli` and `validator` and listed on the root `package.json`.